elasticapm/contrib/serverless/aws.py

```diff
@@ -65,7 +65,7 @@ def handler(event, context):
             return {"statusCode": r.status_code, "body": "Success!"}
     """
 
-    def __init__(self, name=None, **kwargs):
+    def __init__(self, name: str = None, **kwargs) -> None:
```

Comment:
this is type `Optional[str]` if its default value is `None`

Reply:
🤦‍♂️ Honestly don't know how I missed that.